### DISCUSSION TITLE
🟠 Structural Directive | Angular Challenges

### DISCUSSION THREAD

**Discussion by giscus** [tomalaforge/angular-challenges#208] [Announcements]
# 🟠 Structural Directive | Angular Challenges

Challenge 6 is about creating a structural directive to handle permissions

https://angular-challenges.vercel.app/challenges/angular/6-permissions/

**Comment by AbduBehindTheCode** [comment]
I'm not able to start the project with 'npx nx serve permissions'. Can you please check?

**Reply by tomalaforge** [reply]
You are right, sometimes vercel doesn't catch the latest commit. Now should be ok

**Reply by jatingandhi28** [reply]
this is not at above location now
